kartothek/utils/predicate_converter.py

```diff
@@ -0,0 +1,82 @@
+"""
+Helper module to convert kartothek dataset load predicates into cube conditions.
+"""
+
+from typing import Any, List, Sequence, Tuple
+
+import pandas as pd
+import pyarrow as pa
+
+from kartothek.core.cube.conditions import Condition
+
+
+def write_predicate_as_cube_condition(predicate: Tuple[str, str, Any]) -> Condition:
+    """
+    Rewrites a single io.dask.dataset 'read_dataset_as_ddf' predicate condition as cube condition
+
+    Parameters
+    ----------
+    predicate: list
+        list containing single predicate definition
+
+    Returns
+    -------
+    condition: cube condition object
+        cube condition containing the predicate definition
+    """
+    condition_string = None
+    parameter_format_dict = {}
+    if type(predicate[2]) == int:
+        condition_string = "{} {} {}".format(
```

Comment:
I would suggest to use Python f-strings here to be more in line with the rest of the code base